packages/eslint-plugin-office-addins/src/utils/utils.ts

```diff
@@ -72,12 +82,31 @@ function findOfficeApiReferencesInScope(scope: Scope): void {
   scope.childScopes.forEach(findOfficeApiReferencesInScope);
 }
 
+function isMethodFunction(node: TSESTree.MemberExpression): boolean {
+  if (node.parent?.type === AST_NODE_TYPES.CallExpression) {
+    const callExpression: TSESTree.Node = node.parent;
+    if (callExpression.callee === node) {
+      return true;
+    }
+  }
+  return false;
+}
+
+export function isMethodReference(node: TSESTree.Identifier) {
```

Comment:
Did you consider code reuse between these and the load versions of them.  Seems like checking for load should be a special case of checking for a method.

Reply:
I guess it is better to keep them separated for now. The load is considered right now different than the method. 
We would have to change a lot of structures to be able to make load be a special case.

Reply:
I'm not sure I follow changing structures . . . I'm just thinking about the 'isLoadFunction' calling this one instead of the duplicate if statement.  It's not a big deal.